include/roboteam_ai/stp/Play.hpp

```diff
@@ -5,62 +5,106 @@
 #ifndef RTT_PLAY_HPP
 #define RTT_PLAY_HPP
 
+#include <utilities/Dealer.h>
+
 #include <array>
 
-#include "world_new/World.hpp"
 #include "Role.hpp"
+#include "world_new/World.hpp"
 
 namespace rtt::ai::stp {
 
+/**
+ * Play class that's used in the STP model
+ * on update traverses every Role, and updates it.
+ */
+class Play {
+   public:
+    constexpr static size_t ROBOT_COUNT = 11;
```

Comment:
I know I wrote this code but I kinda beuned it, this should be in constants.h tbh

Reply:
Done